src/app/projects/components/wrangler-list-input/wrangler-list-input.component.ts

```diff
@@ -76,27 +67,32 @@ export class WranglerListInputComponent extends BaseInputComponent implements On
     return wrangler ? wrangler.name : '';
   }
 
-  private getWranglerNames(): Observable<string[]> {
-    return this.ingestService.getWranglers().pipe(
-      map(wranglers => wranglers?.map(wrangler => wrangler.name)));
-  }
-
-  private setDefaultWrangler() {
-    // TODO set current account as default wrangler
-  }
-
   private onSearchValueChanged(value: string | Account): Observable<Account[]> {
     const searchText = typeof value === 'string' ? value.toLowerCase() : value.name ? value.name.toLowerCase() : '';
-    return this._filter(searchText);
+    const emptyAccount = new Account({id: null, providerReference: null, name: 'Unassigned'});
+    return this.filterWranglers(searchText).pipe(
+      tap(found_wranglers => found_wranglers.unshift(emptyAccount))
+    );
+  }
+
+  private filterWranglers(wranglerName: string): Observable<Account[]> {
+    const lowerName = wranglerName.toLowerCase();
+    return this.wranglers$.pipe(map(wranglers => wranglers.filter(wrangler => wrangler.name.toLowerCase().includes(lowerName))));
   }
 
-  private _filter(name_or_id: string): Observable<Account[]> {
-    const filterValue = name_or_id.toLowerCase();
+  private findWrangler(accountId: string): Observable<Account> {
     return this.wranglers$.pipe(
-      map(wranglers => wranglers.filter(wrangler => {
-          return wrangler.name.toLowerCase().indexOf(filterValue) === 0 || wrangler.id === name_or_id;
-        })
-      )
+      map(allWranglers => allWranglers.filter(wrangler => wrangler.id === accountId)),
+      map(matchingWranglers => {
+        if (matchingWranglers.length > 0) {
+          if (matchingWranglers.length > 1) {
+            this.alertService.warn('Warning', 'Too many wranglers found for this project.');
```

Comment:
Isn't this warning wrong? The first map returns all of the wranglers with that accountId. So I would have thought it should be "Too many wranglers found for this account". Which also makes no sense. How can there be more than one wrangler for a single accountId?

Reply:
Unless accountId should be projectId

Reply:
I will investigate and confirm.

Reply:
You correct there, changing the error messages to be more useful.

Reply:
I'm also switching the errors to be warnings since you can still save a new wrangler.